packages/cli/api.d.ts

```diff
@@ -0,0 +1 @@
+export * from "./src/api"
```

Comment:
I added this to help IDE and Typescript to see typings from sources. Otherwise everethyng imported from /api was `any`